rest-service/Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN apk add --no-cache tzdata \
 ENV JAVA_HOME=/opt/jdk \
     PATH=/opt/jdk/bin:$PATH \
     XMS=64M \
-    XMX=256M \
+    XMX=1G \
```

Comment:
Bump JVM max memory temporarily until we can resolve the failed queue issue.

Reply:
I'd suggest leaving this to default to 256M and we can set the var higher in prod temporarily.  (unless a 1G default makes sense as a long-term default)

Reply:
Nope, I'd prefer it to remain at 256M, I'll change it back.